index.js

```diff
@@ -28,7 +28,16 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+
```

Comment:
Great work Khaleed!  Remember to add our `;` at the end of statements and lets get into the habit of removing our code comments (even though I love seeing them!)